birdhouse/components/weaver/config/weaver/weaver.ini.template

```diff
@@ -0,0 +1,157 @@
+# enforce provided effective user/group identifiers for Application Package execution
+# values must be resolved as integers
+# (default: use cwltool auto-resolution according to running machine and current user/group)
+weaver.cwl_euid =
+weaver.cwl_egid =
+
+# --- Weaver WPS settings ---
+weaver.wps = true
+weaver.wps_path = ${WEAVER_WPS_PATH}
+weaver.wps_output = true
+weaver.wps_output_dir = ${WEAVER_WPS_OUTPUTS_DIR}
+weaver.wps_output_url = https://${PAVICS_FQDN_PUBLIC}${WEAVER_WPS_OUTPUTS_PATH}
+weaver.wps_output_path =
+weaver.wps_workdir = ${WEAVER_WPS_WORKDIR}
+
+# --- Weaver WPS metadata ---
+weaver.wps_metadata_identification_title=Weaver
+weaver.wps_metadata_identification_abstract=${SERVER_SYNOPSIS}
+weaver.wps_metadata_identification_keywords=Weaver,WPS,OGC,${WEAVER_CONFIG}
+weaver.wps_metadata_identification_accessconstraints=https://${PAVICS_FQDN}/magpie
+weaver.wps_metadata_identification_fees=NONE
+weaver.wps_metadata_provider_name=${SERVER_INSTITUTION}
+weaver.wps_metadata_provider_url=http://pavics-weaver.readthedocs.org/en/latest/
+weaver.wps_metadata_contact_name=${CONTACT_NAME}
+weaver.wps_metadata_contact_position=${CONTACT_POSITION}
+weaver.wps_metadata_contact_phone=${CONTACT_PHONE}
+weaver.wps_metadata_contact_fax=${CONTACT_FAX}
+weaver.wps_metadata_contact_deliveryPoint=${CONTACT_DELIVERY_POINT}
+weaver.wps_metadata_contact_city=${CONTACT_CITY}
+weaver.wps_metadata_contact_stateorprovince=${CONTACT_STATE_OR_PROVINCE}
+weaver.wps_metadata_contact_country=${CONTACT_COUNTRY}
+weaver.wps_metadata_contact_postalcode=${CONTACT_POSTAL_CODE}
+weaver.wps_metadata_contact_email=${SUPPORT_EMAIL}
+weaver.wps_metadata_contact_url=${SUPPORT_URL}
+weaver.wps_metadata_contact_role=${SUPPORT_ROLE}
+
+# --- Weaver WPS REST API ---
+weaver.wps_restapi = true
+weaver.wps_restapi_path = /
+weaver.wps_restapi_ref = https://app.swaggerhub.com/apis/geoprocessing/WPS/
+weaver.wps_restapi_doc = https://raw.githubusercontent.com/opengeospatial/wps-rest-binding/develop/docs/18-062.pdf
+
+# --- Weaver email notification ---
+weaver.wps_email_encrypt_salt = salty-email
+weaver.wps_email_encrypt_rounds = 100000
+weaver.wps_email_notify_smtp_host =
+weaver.wps_email_notify_from_addr = example@email.com
+weaver.wps_email_notify_password = 123456
+weaver.wps_email_notify_port = 25
+weaver.wps_email_notify_ssl = true
+weaver.wps_email_notify_template_dir =
+weaver.wps_email_notify_template_default =
+
+# --- Weaver other configurations ---
+# default WPS processes to always preload
+weaver.wps_processes_file = wps_processes.yml
+# remote ADES dispatching based on data-sources when 'weaver.configuration = EMS'
+weaver.data_sources = data_sources.json
+weaver.extra_options =
+
+###
+# celery scheduler config
+# https://pypi.python.org/pypi/pyramid_celery
+###
+[celery]
+#USE_CELERYCONFIG = True
+BROKER_URL = mongodb://mongodb:27017/celery
+
+###
+# wsgi server configuration
+###
+
+[server:main]
+use = egg:gunicorn#main
+bind = 0.0.0.0:4001
```

Comment:
I've seen this port 4001 a few times.  Should this port be move to `weaver/default.env` so the same value is enforced everywhere?

Reply:
We could do that, but all birds seem to have hardcoded ports all over the place. 
The only case were the value matters (possible conflict) is for exposed ports, which should be fixed by not exposing them at all.